src/index.ts

```diff
@@ -1,17 +1,18 @@
 import shallowEqualArrays, { validArrayValue } from "./arrays";
 import shallowEqualObjects, { validObjectValue } from "./objects";
 
-type Comparable<T> = Record<string, T> | T[] | null | undefined;
+type Comparable = Record<string, any> | any[] | null | undefined;
```

Comment:
Not sure why my editor settings basically decided this... but it fits the spirit, I think.